### ISSUE TITLE
[keyboard navigation] Adjusting in larger steps then smaller steps doesn't work

### ISSUE THREAD

**Issue by JRomero0613** [phetsims/resistance-in-a-wire#160]
Test device: PC Laptop

Operating System: Windows 10 v. 1803

Browser: Chrome v. 67.0.3396.99

Problem description: When adjusting using normal sized steps after adjusting in larger steps the first input does not change the value of the slider as it should. This bug does not occur on the second slider due to the odd increments that it creates when adjusting in larger steps. 

Steps to reproduce: 
1. Navigate to a slider using tab
2. Using pg up/down adjust in larger steps
3. Adjust using normal arrow keys (this must be done in the same direction as you were adjusting with larger steps)
4. The first normal arrow key input will not change the slider value.

<details>
<summary>
Troubleshooting Info:
</summary>
Name: ‪Resistance in a Wire‬
URL: https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.5.0-rc.7/phet/resistance-in-a-wire_all_phet.html
Version: 1.5.0-rc.7 2018-06-28 23:34:38 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36
Language: en-US
Window: 1920x1012
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4095
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"HEAD"},"axon":{"sha":"cc053b4d","branch":"HEAD"},"brand":{"sha":"89d28f63","branch":"HEAD"},"chipper":{"sha":"fe3cb370","branch":"HEAD"},"dot":{"sha":"0dd6ee8e","branch":"HEAD"},"joist":{"sha":"f047fb1b","branch":"HEAD"},"kite":{"sha":"b6071478","branch":"HEAD"},"phet-core":{"sha":"f35ff65e","branch":"HEAD"},"phet-io":{"sha":"d54be499","branch":"HEAD"},"phet-io-website":{"sha":"28284828","branch":"HEAD"},"phet-io-wrapper-classroom-activity":{"sha":"c84e3046","branch":"HEAD"},"phet-io-wrapper-lab-book":{"sha":"ebf7c7dc","branch":"HEAD"},"phet-io-wrappers":{"sha":"ce57c3e2","branch":"HEAD"},"phetcommon":{"sha":"a5a7478c","branch":"HEAD"},"query-string-machine":{"sha":"485e174e","branch":"HEAD"},"resistance-in-a-wire":{"sha":"0520d7c1","branch":"HEAD"},"scenery":{"sha":"33290b5e","branch":"HEAD"},"scenery-phet":{"sha":"42b0cff2","branch":"HEAD"},"sherpa":{"sha":"9c6ec322","branch":"HEAD"},"sun":{"sha":"6a7c923a","branch":"HEAD"},"tandem":{"sha":"7f499d9e","branch":"HEAD"}}
</details>

**Comment by jessegreenberg** [comment]
Hmm, unable to reproduce on master, checking rc.7.

**Comment by JRomero0613** [comment]
It looks like this bug has been fixed in the updated master version, however a new bug that only occurs on Firefox has sprung up in its place. When using tab to navigate between items instead of switching between items each time tab is hit the screen moves down. The screen can not be scrolled back up and the sim becomes unusable. I have tested this on a variety of devices and the bug occurs across all devices only on Firefox.
![riaw_tab](https://user-images.githubusercontent.com/30352536/42349875-526e4d58-806b-11e8-85f4-47689d0c3018.gif)
Here is the error message in the console for Windows 8:
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://phettest.colorado.edu:45362/same-as-remote-master?repo=masses-and-springs. (Reason: CORS request did not succeed).
Here is the error message in the console for Windows 7:
Error: WebGL warning: Failed to create WebGL context: failIfMajorPerformanceCaveat: Driver is not hardware-accelerated.

**Comment by jessegreenberg** [comment]
That looks a lot like https://github.com/phetsims/chipper/issues/687, and probably not related to this issue. This is possibly caused by https://github.com/phetsims/scenery/issues/822, Ill take a look.

**Comment by jessegreenberg** [comment]
@JRomero0613 I expect that is fixed in master, can you please verify? @zepumph I don't think that will impact the RIAW rc.

**Comment by zepumph** [comment]
Sounds good, thanks @jessegreenberg. Focusing back on the main issue, in response to https://github.com/phetsims/resistance-in-a-wire/issues/160#issuecomment-402309941 I tried to cherry-pick https://github.com/phetsims/sun/commit/0ccb537199bef49a6e3dc7090e8a1701e2695741 to a RIAW 1.5 branch of sun, and found a merge conflict. In response I kept:
```
           // format the value text for reading
            var formattedValue = Util.toFixed( value, options.accessibleDecimalPlaces );
            var valueText = StringUtils.fillIn( options.accessibleValuePattern, {
              value: formattedValue
            } );
```
(mainly as a note to self just in case)

I also needed to pull in phetsims/dot@d6eaccb00f34205ac259f693c4d6daf4ea3088e0 to support this onto a dot branch. All is looking good in my local build though, so I'm going to commit and push this (wahoo!).

**Comment by zepumph** [comment]
I think this issue is ready for review in the next RC.

**Comment by JRomero0613** [comment]
@jessegreenberg https://github.com/phetsims/resistance-in-a-wire/issues/160#issuecomment-402864766 is not fixed in master, I have just checked on Windows 10 with Firefox and the screen moves down when tab is hit. It seems to move less than it was previously but the top half of the sim is still inaccessible.

**Comment by jessegreenberg** [comment]
I was able to reproduce this on phettest, but I noticed that common repos were out of date.
![capture](https://user-images.githubusercontent.com/6396244/42387424-74efaee6-80ff-11e8-9994-f9691d74a55d.PNG)

After pressing "Pull All" on phettest I can no longer reproduce. @JRomero0613 can you please verify?

**Comment by JRomero0613** [comment]
@jessegreenberg After pulling all I can no longer reproduce as well, looks like its fixed.

**Comment by jessegreenberg** [comment]
Thanks!

**Comment by zepumph** [comment]
**NOTE: For** QA reviewing this issue for the next RIAW RC, please only confirm that https://github.com/phetsims/resistance-in-a-wire/issues/160#issue-338066478 is fixed in the branch.

**Comment by zepumph** [comment]
@lmulhall-phet in https://github.com/phetsims/QA/issues/137#issuecomment-403938232 you said that QA is done, so I assume that I'm good to close this issue, and that it was successfully checked.

**Comment by zepumph** [comment]
Should be confirmed as part of #179, @KatieWoe please close if it looks good to you.

**Comment by KatieWoe** [comment]
Tested on 1.5.0-rc.8 and 1.5.0-rc.9 on all Win 10 and Mac 10.14 browsers.  The problems described here look ok.  Note that a new issue was found: https://github.com/phetsims/resistance-in-a-wire/issues/179.  Closing this issue.
